src/app/editor/contextualListener.js

```diff
@@ -115,10 +116,16 @@ class ContextualListener {
     let lastCompilationResult = this._deps.compilersArtefacts['__last']
     if (lastCompilationResult && lastCompilationResult.languageversion.indexOf('soljson') === 0) {
       let lineColumn = this._deps.offsetToLineColumnConverter.offsetToLineColumn(position, position.file, lastCompilationResult.getSourceCode().sources, lastCompilationResult.getAsts())
-      let css = 'highlightreference'
+      const css = csjs`
+        .highlightref_fullLine {
+          position:absolute;
```

Comment:
When using absolute, most of the time you want to specify the top / left. At least to make it explicit.

Reply:
did the changes. except this one, anyway ace editor is handling these properties